tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -82,66 +91,19 @@ public void Dispose()
             var spans = agent.WaitForSpans(expectedSpans);
             Assert.Equal(expectedSpans, spans.Count);
 
-            var expectedAppSecEvents = enableSecurity ? 5 : 0;
-            var actualAppSecEvents = 0;
-
-            var spanIds = spans.Select(s => s.SpanId);
-
-            foreach (var span in spans)
-            {
-                // not all tags will have security events, only the route ones
-                var gotTag = span.Tags.TryGetValue(Tags.AppSecJson, out var json);
-                if (gotTag)
-                {
-                    actualAppSecEvents++;
-
-                    // we only really care about these asserts if we're on an appsec span
-                    foreach (var assert in assertOnSpans)
-                    {
-                        assert(span);
-                    }
-
-                    var jsonObj = JsonConvert.DeserializeObject<AppSecJson>(json);
-
-                    var item = jsonObj.Triggers.FirstOrDefault();
-                    Assert.NotNull(item);
-
-                    var attackEvent = item;
-                    var shouldBlock = expectedStatusCode == HttpStatusCode.Forbidden;
-                    Assert.Equal("Finds basic MongoDB SQL injection attempts", attackEvent.Rule.Name);
-
-                    // presences of json tag implies span should carry other security tags
-                    var securityTags = new Dictionary<string, string>()
-                    {
-                        { "appsec.event", "true" },
-                        { "_dd.origin", "appsec" },
-                        { "http.useragent", "Mistake Not..." },
-                        { "actor.ip", "86.242.244.246" },
-                        { "http.request.headers.host", $"localhost:{_httpPort}" },
-                        { "http.request.headers.x-forwarded", "86.242.244.246" },
-                        { "http.request.headers.user-agent", "Mistake Not..." },
-                    };
-                    foreach (var kvp in securityTags)
-                    {
-                        Assert.True(span.Tags.TryGetValue(kvp.Key, out var tagValue), $"The tag {kvp.Key} was not found");
-                        Assert.Equal(kvp.Value, tagValue);
-                    }
-                }
-            }
-
-            // asserts on request status code
-            Assert.All(resultRequests, r => Assert.Equal(r.StatusCode, expectedStatusCode));
-
-            // asserts on the security events
-            Assert.Equal(expectedAppSecEvents, actualAppSecEvents);
+            // Overriding the type name here as we have multiple test classes in the file
+            // Ensures that we get nice file nesting in Solution Explorer
+            await Verifier.Verify(spans, settings)
+                          .UseMethodName("_")
+                          .UseTypeName(GetTestName());
```

Comment:
Given 90% of this PR is all to use this helper, I'm strongly inclined to just link the `VerifyHelper.cs` file instead of creating a separate helper project.

Reply:
Okay, done removed the commits via a rebase.